README.md

```diff
@@ -87,6 +87,21 @@ Make sure that the sink's formatter outputs enriched properties. `Serilog.Sinks.
 .WriteTo.Console(outputTemplate: "{Timestamp:yyyy-MM-dd HH:mm:ss.fff zzz} [{Level:u3}] {Message:lj}{NewLine}{Exception} {Properties:j}")
 ```
 
+### JSON `appSettings.json` configuration
+
+Alternatively to fluent configuration seeting can be stored in `appSettings.json` using [_Serilog.Settings.Configuration_](https://github.com/serilog/serilog-settings-configuration):

```

Comment:
Typo in `seeting`.

Reply:
Additionally, I would just say `can be stored in application configuration`. `appsettings.json` is not treated any differently so no need to mention it specifically.

Reply:
Typo fixed. Right, just looked at other repositories for example, but this is totally legit.